lib/find.js

```diff
@@ -30,7 +30,7 @@ function pointFinder(name) {
 
             return data.Points.filter(function (p) {
                 p = p.split('|');
-                return p[2].length === 0 || p[2] === 'Dresden';
+                return p[3] && (p[2].length === 0 || p[2] === 'Dresden');
```

Comment:
ok you are absolutely right. Did you find that special case by accident? All other requests are reposting an invalid status or no points. That might be a bug, the id for '0' finds Velveta 03 in Varnsdorf.